styles.css

```diff
@@ -330,6 +331,19 @@ ul {
   width: 327px;
 }
 
+.Get_btn button {
+  margin-top: 222px;
+  height: 50px;
+  width: 108px;
```

Comment:
- [ ] Kindly make sure the `button` in your Contact Me section looks as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/207445704-a9f27029-57aa-4c33-aef9-7b4f91288524.png)  |  ![image](https://user-images.githubusercontent.com/74814780/207445569-12e3564a-b734-427c-9016-abb8fd1b52c2.png)